drupal/config/sync/block.block.breadcrumbs.yml

```diff
@@ -28,14 +28,6 @@ visibility:
     bundles:
       digital_object: digital_object
       digital_object_collection: digital_object_collection
-  node_type:
```

Comment:
Wait! The replacement is missing here? I see the node_type gone but not new new Conditional?

Reply:
Or am I missing something?

Reply:
@DiegoPino Unless I'm misunderstanding, I'm assuming that the `'entity_bundle:node'` just above it is the correct one since replacing just as the update hook does would make them identical, but I wanted to check in a deployment just to make sure I'm not missing something.

Reply:
@aksm No worries, I was mistaken. I saw a bunch of lines removed and did not notice the good/new ones were already present. I will merge this